modules/ROOT/pages/import-api-task.adoc

```diff
@@ -6,10 +6,10 @@ endif::[]
 You can develop an API for and export it from API Manager for future use or for sharing. The exported API can be imported using this procedure. To perform this task, you need to have a ZIP file on your file system containing an API object exported from API Manager.
 
 . After logging into Anypoint Platform, click API Manager.
-. In API Administration, select Manager API > Import API from the Desktop.
+. In API Administration, select *Manager API > Import API from the Deskto*p.
```

Comment:
```suggestion
. In API Administration, select *Manager API > Import API from the Desktop*.
```

Reply:
Because of the misplaced format symbol *, the following steps appear here in bold. Please verify that this does not carry over in the production version.

Reply:
Done! Thanks!